dt/DTs_Designate/DFG-network-openstack-designate-17.1_director-rhel-virthost-3cont_2comp-ipv6-geneve.md

```diff
@@ -0,0 +1,64 @@
+# Deployed Topology 3_ocp_workers_2_compute_ipv6_geneve
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change              |  Date   |
+| -------: | :------------------ | :-----: |
+|     v0.1 | Initial publication | 2023-12-12 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Networker nodes                            | vm    | 3      |
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+|-----------|------------------------------| ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+| Glance    | Ceph/RBD                     | Interchangable     |
```

Comment:
Please check the last version of https://github.com/openstack-k8s-operators/architecture/pull/38/files#diff-b82405d52f27c2965f7f00073f14cc6642b521dfc618231689c2d3c6943e41e9  as it suggest a few more fields, like

- a description field which explain the "why" of the topology, so that it's easy for people checking it to understand the rationale for the choice of components
- a separate table for the components which needs to be there but where you don't really care about the configuration (which in this case may be all the storage ones, where you don't really need ceph, maybebut given the lack of the description it's not really clear). For example it's not clear why keystone requires a special configuration, does it designate require it? Or are you combining it with some require configuration for security?